code/modules/tgui/modules/crew_monitor.dm

```diff
@@ -1,9 +1,22 @@
+#define MIN_ZOOM 1
+#define MAX_ZOOM 8
+#define MIN_TAB_INDEX 0
+#define MAX_TAB_INDEX 1
+
 /datum/ui_module/crew_monitor
 	name = "Crew monitor"
 	var/is_advanced = FALSE
 	var/viewing_current_z_level
 	/// If true, we'll see everyone, regardless of their suit sensors.
 	var/ignore_sensors = FALSE
+	var/tab_index = 0
```

Comment:
My only issue with this is that 2 people using the UI will result in fighting over tabs and zoom.

Then again that's how it would be if this were an actual computer.